src/test/java/io/personium/test/jersey/cell/auth/AuthAccountStatusTest.java

```diff
@@ -128,9 +128,9 @@ public final void test_handlePassword() {
         String url = (String) responseBody.get("url");
         String lastAuthenticated = (String) responseBody.get("last_authenticated");
         Long failedCount = (Long) responseBody.get("failed_count");
-        assertThat(error, is(OAuth2Helper.Error.UNAUTHORIZED_CLIENT));
+        assertThat(error, is(OAuth2Helper.Error.INVALID_GRANT));
```

Comment:
Changed the error reason.
When password needs change, the client does not have any fault,
rather we could interpret like user's grant is not sufficient.